DigitalLearningSolutions.Data.Tests/DataServices/CentresDataServiceTests.cs

```diff
@@ -2,10 +2,12 @@
 {

```

Comment:
Could we add the Centre configuration + Edit manager detail pages to the automated accessibility tests?